sdks/wasm/Mono.WebAssembly.DebuggerProxy/DebugStore.cs

```diff
@@ -743,19 +743,21 @@ static bool Match (SequencePoint sp, SourceLocation start, SourceLocation end)
 			if (start.Column > spEnd.Column && start.Line == spEnd.Line)
 				return false;
 
-			if (end.Line < spStart.Line)
-				return false;
+			if (end != null) {
+				if (end.Line < spStart.Line)
+					return false;
 
-			if (end.Column < spStart.Column && end.Line == spStart.Line)
-				return false;
+				if (end.Column < spStart.Column && end.Line == spStart.Line)
+					return false;
+			}
 
 			return true;
 		}
 
 		public List<SourceLocation> FindPossibleBreakpoints (SourceLocation start, SourceLocation end)
 		{
 			//XXX FIXME no idea what todo with locations on different files
-			if (start.Id != end.Id) {
+			if (end != null && start.Id != end.Id) {
```

Comment:
Won't `end != null` always be true? 
I only found that `FindPossibleBreakpoints` is called by `GetPossibleBreakpoints` under [MonoProxy.cs](https://github.com/mono/mono/blob/a1b68d97c3ef261f4aeede766e8221dbdf33796e/sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs#L187), and it looks like `GetPossibleBreakpoints` is executed when `end != null` is true [here](https://github.com/mono/mono/blob/a1b68d97c3ef261f4aeede766e8221dbdf33796e/sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs#L187)

Reply:
Oops, thanks for pointing that out - I forgot to also change `MonoProxy.cs`.

Reply:
According to the DevTools Protocol, `start` is required but `end` is optional.
https://chromedevtools.github.io/devtools-protocol/1-3/Debugger/#method-getPossibleBreakpoints

I have fixed the call site - thank you so much for pointing that out.

Reply:
Ah I see, that makes sense :)